book/plugins/general-abilities/multiple-types-one-repo/multiple-types-one-repo.md

```diff
@@ -0,0 +1,37 @@
+# Using One Code Base for Multiple Plugins
+
+A developer can use one codebase for handling different plugin types.
+
+The main difference between [plugin types](/getting-started/plugin-types.md) and [plugin abilities](/getting-started/plugin-abilities.md) is the fact a single plugin can implement multiple abilities but can be of one type.
+
+For example, you can build a push plugin and later add support for an analytics plugin in the same codebase.
```

Comment:
this literally contradicts the previous sentence - if you don't add an explanation - so 
you can build a library that implements a push plugin and later use the same library to add implementation for an analytics plugin.

you see the difference right?

Reply:
yep.
I removed it.